### PR TITLE
November 2018 updates to README

### PR DESCRIPTION
**Pull request by adutko**
Update the readme to be more concise, and to provide additional clarity around steps that were confusing during my initial project set up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,35 +43,34 @@ vagrant ssh
...
 
-1. Add photo(s) (with faces, this is about emotions, after all!) into your in directory (by default, `site/in`). This can be done automatically by setting this folder as the directory where a digital camera outputs photos.
-2. Go to the [Keep/Kill](http://localhost:8080/buttons) page. To process the session, click on "Keep". To cancel session, click on "Kill". You will need to add more photos if you do this.
-3. Watch the [panel](http://localhost:8080) and [grid](http://localhost:8080?showgrid) pages to see the output.
+1. Add photo(s) (with faces, this is about emotions, after all!) into your in directory (by default, `site/in`). This can be done automatically by setting this folder as the directory where a digital camera outputs photos. You can manually add photos to this folder as well, by simply dragging and dropping them in. Please note that all images in `/in` must have _unique file names_ to trigger the API. 
+2. Go to the [Keep/Kill](http://localhost:8080/buttons) (`localhost:8080/buttons`) page. To process the session, click on "Keep". To cancel session, click on "Kill" (you will need to add more photos if you kill the session.)
```
Comment (paulfarning) [inline README.md]: Period should be outside the closing paren.

File: README.md [hunk 2]
```diff
@@ -77,20 +80,20 @@ This experience can post to [Twitter](https://twitter.com/GCPEmotobooth/status/7
...
-  - `timing=noFace&`: all face-related steps are skipped; only aura animates in.
-  - `timing=noAura&`: all face-related steps run, and then the animation stops.
-  - `timing=noChrome&`: all face and aura animations run, but chrome does not render.
-- Event options: Emotobooth supports multiple designs for different events, these are added and handled inside of main.js, here are the currently supported events
+  - `?timing=fast&`: all steps run, but faster than usual.
+  - `?timing=finalOnly&`: all steps are skipped; only final aura is rendered.
+  - `?timing=noFace&`: all face-related steps are skipped; only aura animates in.
+  - `?timing=noAura&`: all face-related steps run, and then the animation stops.
+  - `?timing=noChrome&`: all face and aura animations run, but chrome does not render.
+- Event options: Emotobooth supports multiple designs for different events (these are added and handled inside of main.js.) Some events include:
```
Comment (paulfarning) [inline README.md]: Period should be outside the closing paren.

File: README.md [hunk 3]
```diff
@@ -77,20 +80,20 @@ This experience can post to [Twitter](https://twitter.com/GCPEmotobooth/status/7
...
+There are several querystrings that can be added to the grid page's url to affect its appearance and behavior. Default behavior (what happens without a querystring) is what we would want in a production environment.
 
-- `showgrid`: **the exception; we will want to use this on one of the production screens** shows the grid / historical view rather than the json / latest view.
-- `zoom`: resize UI so panels can fit, based on width, in your window.
-- `controls`: in json view, adds toolbar with shortcuts to taking certain photos to bottom of screen
-- `dontprint`: tells the application not to render final photostrip to the `site/out-print` folder
-- `prepopulate`: in grid view, uses pregenerated images to fill out grid if we don't actually have enough historical images to do so. Otherwise, only historical processed images will be used.
+- `?showgrid`: **the exception; we will want to use this on one of the production screens** shows the grid / historical view rather than the json / latest view.
```
Comment (paulfarning) [inline README.md]: I see how hinting at the query string question mark prefix could be helpful but also seems somewhat misleading as this is a list of parameters names available. Can more than one param be applied at a time? In that case, the url would be, for example, `http://localhost:8080?showgrid&controls&timing=fast`, not `http://localhost:8080?showgrid?controls?timing=fast`.

Maybe approach would be to indicate a query string can be added to the base url with multiple (assuming that is true) params from the list below and then provide an example url for reference?

Comment (adutko) [reply README.md]: @paulfarning Thanks for the review! I just fixed the punctuation issues and updated the query string section to try to make it more clear - let me know what you think.

Comment (adutko) [reply README.md]: Oh duh - just looked at your comment again. Yes, I was incorrectly adding these with `?` between them, but I see now that the and `&` symbol is the correct was to pass multiple parameters to a query string - I will update. 👍 `http://localhost:8080/?showgrid&prepopulate&zoom`

**Review by paulfarning — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**